samples/SuperObject.java

```diff
@@ -20,6 +20,11 @@
 @NullMarked
 class SuperObject {
   void foo(
+      // Wildcards with `? super Object` must have the same upper and lower bound. Here, the upper
+      // bound is @Nullable, whereas the lower bound is @NonNull.
+      // As javac doesn't allow us to keep track of the separate annotations, there is an error
+      // here.
+      // :: error: jspecify_conflicting_annotations
```

Comment:
I need to add a mapping to `(type.invalid.super.wildcard) bounds must have the same annotations.` in a separate PR against the reference checker.

Reply:
https://github.com/jspecify/jspecify-reference-checker/pull/166